src/gui/aboutdialog.ui

```diff
@@ -57,7 +57,7 @@
    <item>
     <widget class="QLabel" name="versionLabel">
      <property name="text">
-      <string>Version 0.2.17</string>
+      <string>Version</string>
```

Comment:
Have you check how does it work with Russian translation? Does word version correctly displayed in Russian?

Reply:
fixed